src/mumble/MainWindow.cpp

```diff
@@ -3076,24 +3076,27 @@ void MainWindow::trayAboutToShow() {
 		p = QCursor::pos();
 	}
 
-	QRect qr = Screen::screenAt(p)->geometry();
-
-	if (p.y() < (qr.height() / 2))
-		top = true;
-
-	qmTray->clear();
-	if (top) {
-		qmTray->addAction(qaQuit);
-		qmTray->addAction(qaShow);
-		qmTray->addSeparator();
-		qmTray->addAction(qaAudioDeaf);
-		qmTray->addAction(qaAudioMute);
-	} else {
-		qmTray->addAction(qaAudioMute);
-		qmTray->addAction(qaAudioDeaf);
-		qmTray->addSeparator();
-		qmTray->addAction(qaShow);
-		qmTray->addAction(qaQuit);
+	QScreen *screen = Screen::screenAt(p);
+	if(screen != NULL){
```

Comment:
`if (screen != nullptr) {`

Reply:
Do all our target systems support nullptr? I think we have to or at least definitely in the past had to use NULL.

Reply:
We always used `NULL`, for consistency.

The issue is that very recent compilers print a warning for each use of `NULL` instead of `nullptr`.

`nullptr` was introduced in C++11, which is our minimum requirement starting with 1.4; Qt is replacing `Q_NULLPTR` (macro defined to either `nullptr` or `NULL`, depending on the compiler) with `nullptr` everywhere: https://bugreports.qt.io/browse/QTBUG-52736

Reply:
We should update our coding guidelines then, and preferably have some kind of documentation on what we change for 1.4 - maybe start the 1.4 wiki page with a code section. And if we want to switch we should have a ticket to replace it across the code base.

Reply:
I agree.